internal/subscriptions.go

```diff
@@ -76,7 +77,11 @@ func requestRegcodes(data SUSEConnectData, credentials Credentials) ([]string, e
 	}
 
 	for _, subscription := range subscriptions {
-		codes = append(codes, subscription.RegCode)
+		if subscription.Status != "EXPIRED" {
```

Comment:
Maybe:

```go
if strings.ToUpper(subscription.Status) != "EXPIRED" {
```

Just in case there are no funny business with the exact string.

Reply:
added commit for that